addons/base_action_rule/base_action_rule.py

```diff
@@ -194,7 +194,12 @@ def _process(self, records):
 
         # subscribe followers
         if self.act_followers and hasattr(records, 'message_subscribe'):
-            records.message_subscribe(self.act_followers.ids)
+            followers = self.env['mail.followers'].sudo().search([
+            ('res_model', '=', records._name), ('res_id', 'in', records.ids),
+            ('partner_id', 'in', self.act_followers.ids)])
+            from ipdb import set_trace; set_trace()
+            if not len(followers) == len(self.act_followers):
+                records.message_subscribe(self.act_followers.ids)
```

Comment:
Should it be `message_subscribe()` the one with the responsibility to ensure that there are no duplicate followers? Maybe I'm remembering wrong, but I believe that it did that in previous versions.